packages/galata/src/galata.ts

```diff
@@ -1518,6 +1598,30 @@ namespace galata {
         // go to home folder
         await filebrowser.openHomeDirectory();
     }
+
+    export
+    async function isInSimpleMode(): Promise<boolean> {
```

Comment:
An alternative could be to access `shell.mode` which should be either `single-document` or `multiple-document`?

Reply:
Thanks @jtpio I liked this idea and tried to replace it. However, `mode` is not exposed with `IShell` (`window.jupyterlab.shell` which I have access to). I could cast the type but not sure if that is ideal.

Reply:
Ah right that's for an `ILabShell`. Thanks for checking :+1: 

Was mostly curious whether this could have made the code even more canonical, but the current approach also looks good.